wallet/src/util/WalletBackendAdapter.ts

```diff
@@ -229,9 +229,14 @@ export const makeWalletBridgeFromFollowers = (
       watcher.watchLatest(
         [
           AgoricChainStoragePathKind.Data,
-          `published.beansOwing.${smartWalletKey.address}`,
+          `beansOwing.${smartWalletKey.address}`,
         ],
-        value => beansOwingUpdater.updateState(Number(value)),
+        value => {
+          beansOwingUpdater.updateState(Number(value));
+        },
+        err => {
+          console.error('error watching beansOwing', err);
```

Comment:
>Should watchLatest require an error handler?

I think optional is okay, we don't need to mandate that for all consumers

>Shouldn't we alert the user to errors like this? What other failures are being ignored like this was?

Good point, I guess it makes sense to do a little more than just log to console (which was the previous behavior). We have an `errorHandler` hook that we're passing in which shows a toast and sets the wallet state to an error, hiding purses and stuff. Now that I think about it, it's probably not overkill to do so, because if something went wrong in any part it's probably a big enough deal. Just updated the adapter to call that hook in all cases.